### PR TITLE
[READY] Adds the Golem Commander bundle.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus, sprites by Kryson
add: Golem commander bundle, a type B bundle that gives you ample tools to create your own army.
add: Syndicate Asset register, allows you to rename golems
add: Stimgun, Fire this at your golems to charge them up with a new chemical that makes them faster and stronger. It heats up other races due to the heating nature of the drug.
/:cl:

the stimgun has a limit on how much you can fill up a mob with at a time, slowly decreasing in amount of chems it adds until it locks up at 8.

### CODE REVIEW

**Comment by Qustinnus** [comment]
also im back from hiatus

**Comment by adamkad1** [comment]
why not just make it a commie golem conversion item

**Comment by Qustinnus** [comment]
because thats admin meme stuff

**Comment by wesoda25** [comment]
Epi c

**Comment by necromanceranne** [comment]
Now to get them all into fedoras and make it golem mobsters

**Comment by Shadowflame909** [comment]
I'd give it 3 adamantine slimes.


Otherwise i'd have to break into xenobio or get a job change to use this

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -914,17 +914,33 @@
...
+		L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown = -2.5, blacklisted_movetypes=(FLYING|FLOATING))
+	else
+		L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown = -0.75, blacklisted_movetypes=(FLYING|FLOATING))
+
+/datum/reagent/drug/fervoxazine/on_mob_life(mob/living/carbon/M)   
+	if(prob(5))
+		var/high_message = pick("You feel heated and ready to move.", "You feel like a burning star.", "You feel like a meteor.")
+		to_chat(M, "<span class='notice'>[high_message]</span>")
+	M.adjust_bodytemperature(BODYTEMP_HEATING_MAX) //hot hot hot
+	M.adjustStaminaLoss(-2, 0)
```
Comment (Tlaltecuhtli) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: -2 stamina/tick :joy: :joy: :joy:

**Comment by Tlaltecuhtli** [comment]
may be really epic when you make cardboard golems ahahah

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: its a meth gun for golems i dont know what you expected

Comment (Tlaltecuhtli) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: all the antistun chems have outdated staminaheal as -2 is even less than natural regen

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 2]
```diff
@@ -914,17 +914,33 @@
 	..()
 	. = 1
 
-/datum/reagent/medicine/stimulants
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Wait, did you just remove normal stimulants from the game?

Comment (Qustinnus) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: lol accident ill fix that

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: ill fix

**Comment by Cyprex** [comment]
This just seems op as-is, make it 2 golems and remove the bible+bolas

**Comment by Qustinnus** [comment]
no

**Comment by davethwave** [comment]
The problem with removing the other stuff Cyprex is the reliability of the golems you may get.

**Comment by Cyprex** [comment]
Some decent-ish gear for you vs 3 golems+ a gun to make them better, do you see the difference here?

**Comment by ATH1909** [comment]
Yeah, this DOES kind of seem a bit over the top. Drop eveything but the stim gun, the megaphone, the bible, and the golem shells, and it should be fine.

**Comment by Qustinnus** [comment]
it doesnt seem that OP to me at all, youre risking all your TC and rolling in an unreliable table of loot

**Comment by ATH1909** [comment]
> it doesnt seem that OP to me at all, youre risking all your TC and rolling in an unreliable table of loot

I meant in comparison to the other type B bundles.

**Comment by Qustinnus** [comment]
yeah but they suck balls in general and arent that fun to use

**Comment by Shadowflame909** [comment]
thats true but this also sucks


It gives you all the gear FOR golems. But you get no shells



Am I expected to do 1 hour of xenobiology for this gimmick

**Comment by Cyprex** [comment]
You get adamantine @Shadowflame909

**Comment by Shadowflame909** [comment]
dont you just put that into shells??

**Comment by davethwave** [comment]
Ever heard of waiting for mats, or you know... raiding some place for a stack of metal? Mats aren't that rare.

**Comment by Shadowflame909** [comment]
What's the point of mats if you don't have shells 🤔

**Comment by Borisvanmemes** [comment]
Adamantine can be used to make golem shells iirc.

**Comment by Shadowflame909** [comment]
Oh.



Mega oof i apologize to mr qustinnus good pr




I think this PR fits since all the Special items so far are useless. This gives you 3 servants that are temporarily at the same speed as humans. That's neat

**Comment by wesoda25** [comment]
Yeah this gives reason to roll special B unless someone wanted a ninja flavor murderbone

**Comment by optimumtact** [comment]
it's a low quality idea

**Comment by ExcessiveUseOfCobblestone** [comment]
Status?

**Comment by Qustinnus** [comment]
doing some final balancing because body temps is being dumb

**Comment by ExcessiveUseOfCobblestone** [comment]
When you're around my body temp goes up if you know what i'm sayin'

**Comment by Qustinnus** [comment]
when you're around more than just my body temp goes up if you know what im saying

**Comment by optimumtact** [comment]
still bad

**Comment by Qustinnus** [comment]
whats bad about it i cant really change anything about it based on a 2 word comment

**Comment by optimumtact** [comment]
golems are lame

**Comment by Qustinnus** [comment]
yeah but some ppl like golems so this is fun for them

**Comment by optimumtact** [comment]
thog ambivalent, thog leave for other maintainer

**Comment by SpaceManiac** [comment]
Is this still a WIP or are you waiting for feedback?

**Comment by Qustinnus** [comment]
Mmmh..the only thing thats WIP is the amount of heat stimulant gives, besides that its ready.

**Comment by Qustinnus** [comment]
@optimumtact this isn't stale it just needs a review :(

**Comment by optimumtact** [comment]
yes and you haven't managed to find someone, and I aint doing it, so it's stale

**Comment by Qustinnus** [comment]
Well who the hell else can I ask for review? I dmed people but review begging is frowned upon

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm assuming no one wants it but can't be arsed to just say so.

**Pull request closed without merging**